db/migrate/20230606182546_add_likes_counter_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddLikesCounterToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :Likes_counter, :integer
```

Comment:
[OPTIONAL] It would be nice if you make :LIkes_counter in lowercase because it my cause you some issues in future use. 

```suggestion
    add_column :posts, :likes_counter, :integer
```

Reply:
Thanks for the insights @Rustamxon7, i will fix it 😉